File: php/dbconnect.php
<?php

function dbConnect() {
  $db = new PDO(
	  'mysql:host=localhost;dbname=crm;port=3307;charset=utf8',
      'root',
      ''
  );
  $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
  $db->exec('SET NAMES utf8');
  return $db;
}

File: backend/dbconnect.py
import mysql.connector

def get_connection():
    return mysql.connector.connect(
        host="127.0.0.1",
        port=3307,
        user="root",
        password="",
        database="crm",
        charset="utf8mb4",
        collation="utf8mb4_general_ci"
    )